eks/terraform.tfvars

```diff
@@ -2,8 +2,53 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Sample tfvars file. Uncomment out values to use
+# Do not commit this file to Git with sensitive values
+
 # cluster_name = "holoscan-cluster"
 # region  = "us-west-2"
 
 # Optional: If deploying into an existing VPC, use the following variable
```

Comment:
Probably this comment should be moved just above `existing_vpc_details` variable.

Reply:
Fixed in #11